### ISSUE TITLE
gen_src_tree not set up right (?)

### ISSUE THREAD

**Issue by evmar** [kaizouman/build-benchmark#1]
```
sigint:~/projects/build-benchmark$ make
Makefile:8: *** Please launch gen_src_tree.sh first.  Stop.
sigint:~/projects/build-benchmark$ ./gen_src_tree.sh 
sigint:~/projects/build-benchmark$ make
Makefile:8: *** Please launch gen_src_tree.sh first.  Stop.
```

**Comment by dacorvo** [comment]
Hi,
My mistake: this is poorly documented.
You need to specify three parameters to the script: number of directories at each stage, tree depth, and output directory.

./gen_src_tree.sh 10 4 ./output/src

Then you should choose the build system you want to test:

time make ninja -j16 

The Makefile expects the sources to be under ./output/src by default, but you can pass a different source directory on the command line.

SRC=/path/to/sources time make ninja -j16 

If you want to run the full benchmark, launch ./benchmark.sh

**Comment by dacorvo** [comment]
Hmmm ... the default value for the output directory was wrong in gen_src_tree.sh: will fix it.

**Comment by dacorvo** [comment]
Fixed in 375e0345c1b7ded8a30cb1649f41f2653594a748
